janusgraph-codepipelines-ci/src/main/java/org/janusgraph/codepipelines/AwsCodePipelinesCi.java

```diff
@@ -0,0 +1,290 @@
+package org.janusgraph.codepipelines;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.profile.ProfileCredentialsProvider;
+import com.amazonaws.services.codebuild.AWSCodeBuild;
+import com.amazonaws.services.codebuild.AWSCodeBuildClientBuilder;
+import com.amazonaws.services.codebuild.model.*;
+import com.amazonaws.services.codepipeline.AWSCodePipeline;
+import com.amazonaws.services.codepipeline.AWSCodePipelineClientBuilder;
+import com.amazonaws.services.codepipeline.model.*;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.identitymanagement.model.*;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.BucketTaggingConfiguration;
+import com.amazonaws.services.s3.model.TagSet;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import org.apache.commons.cli.*;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Created by amcp on 2017/02/20.
+ */
+public class AwsCodePipelinesCi {
+    public static final String EFFECT = "Effect";
+    public static final String RESOURCE = "Resource";
+    public static final String ACTION = "Action";
+    public static final String VERSION = "Version";
+    public static final String VERSION_DATE = "2012-10-17";
+    public static final String STATEMENT = "Statement";
+    public static final String AWS_CODEBUILD_JAVA_OPENJDK_8 = "aws/codebuild/java:openjdk-8";
```

Comment:
And maybe we want to run with a different JDK or environment. Anyway, ill pull it out.